.golangci.yml

```diff
@@ -3,30 +3,31 @@ linters-settings:
     locale: US
 
 linters:
-  # please, do not use `enable-all`: it's deprecated and will be removed soon.
-  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
-  # disable-all: true
   enable:
```

Comment:
Have others that I'd like to add but require some work that isn't priority now... more in the future.